### DISCUSSION TITLE
Previous Builder Versions

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#6894] [Help] [answered]
Since the SQL database issue remains #6846, I'm hoping to install a previous version such and see if i can workaround the issue.
My concern is, if i move my applications to a previous version of budibase, will it generate conflicts?

Also, what version would you suggest? I'm using docker-composer. 

Thanks!

**Comment by melohagan** [comment, marked as answer]
Hi @insomniacj93 

The previous v1.0 version is 1.0.218. If you do roll back Budibase, you would also need to roll back the version of your app to 1.0.218 as well.

**Reply by insomniacj93** [reply]
Hi @melohagan 
Since I've started from a fresh new install, i can no longer roll back to previous installed versions.
Is there any way i can install the previous version through cmd commands?

**Reply by melohagan** [reply]
Hey @insomniacj93 You can follow the steps outlined here: https://github.com/Budibase/budibase/discussions/6353
